projects/kit/components/input-phone-international/input-phone-international.options.ts

```diff
@@ -0,0 +1,38 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk';
+import type {TuiCountryIsoCode} from '@taiga-ui/i18n';
+import type {MetadataJson} from 'libphonenumber-js/core';
+import type {Observable} from 'rxjs';
+import {defer} from 'rxjs';
+
+export interface TuiInputPhoneInternationalOptions {
+    readonly countries: readonly TuiCountryIsoCode[];
+    readonly countryIsoCode: TuiCountryIsoCode;
+    readonly metadata: Observable<MetadataJson>;
+}
+
+export const TUI_INPUT_PHONE_INTERNATIONAL_DEFAULT_OPTIONS: TuiInputPhoneInternationalOptions =
+    {
+        countries: [],
+        countryIsoCode: 'RU',
+        metadata: defer(async () =>
+            import('libphonenumber-js/max/metadata').then(m => m.default),
```

Comment:
New options are almost identical to the previous [one](https://github.com/taiga-family/taiga-ui/blob/main/projects/legacy/components/input-phone-international/input-phone-international.options.ts).

`metadata` is the only addition to it.

Reply:
Maybe it is better to set default metadata `libphonenumber-js/min` ? I think it should cover 90% of use cases. 
(and show in "setup" page or in some example that we can provide our own metadata)

Reply:
> Maybe it is better to set default metadata libphonenumber-js/min ?

Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/a81ff3547f6258624a0c5836b8942548ecdd2ee9


> and show in "setup" page or in some example that we can provide our own metadata

I will rebuild the whole documentation page of `PhoneInternational` in a separate PR later.